src/scripts/core/core_pending_purposes.js

```diff
@@ -0,0 +1,36 @@
+import { logInfo } from './core_log';
+
+export let pendingPurposes = false;
+
+export function getPendingPurposes() {
+  return pendingPurposes;
+}
+
+export function setPendingPurposes(purposes) {
+  pendingPurposes = purposes;
+  logInfo('set pending purposes', pendingPurposes)
+}
+
+export function addPendingPurpose(Id) {
+  const id = parseInt(Id, 10);
```

Comment:
Little bit confusing naming, just a different in case for one letter

Reply:
good point, added suggestive naming instead